thirdparty/runtime_environments.go

```diff
@@ -58,3 +65,33 @@ func (c *RuntimeEnvironmentsClient) getImageNames(ctx context.Context) ([]string
 	}
 	return filteredImages, nil
 }
+
+// getPackages returns a list of package changes (name and version) from the corresponding AMI id.
+func (c *RuntimeEnvironmentsClient) getPackages(ctx context.Context, amiID string, page string, limit string) ([]Package, error) {
```

Comment:
This function also needs to support filters for the `name` & `manager` fields. Since there are a lot of input parameters, you might want to turn them into a struct. Here's an [example](https://github.com/evergreen-ci/evergreen/blob/c52ac012645fe6700b87453b6db94d031737b879/model/host/host.go#L3274-L3285) of that